packages/typespec-ts/test/modularUnit/anonymousModel.spec.ts

```diff
@@ -364,7 +364,8 @@ describe("anonymous model", () => {
         );
       });
 
-      it("should not flatten model if spread in the payload with required parameters", async () => {
+      // TODO: Re-enable when related work is merged
+      it.skip("should not flatten model if spread in the payload with required parameters", async () => {
```

Comment:
I am not quite sure why this is failing now? I didn't see we have done any specific change for it yet?

Reply:
yup I was testing the latest version and this was a leftover